backend/migrations/versions/b833ad41db68_maintenance_window_schema.py

```diff
@@ -11,15 +11,18 @@
 from alembic import op
 import sqlalchemy as sa
 from sqlalchemy import Column, String, orm
+from sqlalchemy.ext.declarative import declarative_base
 
-from dataall.base.db import get_engine, has_table, Base
+from dataall.base.db import get_engine, has_table
 
 # revision identifiers, used by Alembic.
 revision = 'b833ad41db68'
 down_revision = '458572580709'
 branch_labels = None
 depends_on = None
 
+Base = declarative_base()
```

Comment:
irrelevant change but it was preventing alembic from loading because the `maintenance` table was trying to be defined twice. Following the pattern of the rest of the alembic script we redefine base instead of reusing.